onmt/utils/Opt.lua

```diff
@@ -1,21 +1,3 @@
-local function isSet(opt, name)
-  return opt[name]:len() > 0
-end
-
---[[ Check that option `name` is set in `opt`. Throw an error if not set. ]]
-local function requireOption(opt, name)
-  if not isSet(opt, name) then
-    error("option -" .. name .. " is required")
-  end
-end
-
---[[ Make sure all options in `names` are set in `opt`. ]]
-local function requireOptions(opt, names)
-  for i = 1, #names do
-    requireOption(opt, names[i])
-  end
-end
-
 --[[ Convert `val` string to its actual type (boolean, number or string). ]]
```

Comment:
yes - I moved the reading and saving functions to extendedCmdLine - we now have by default:
`-config FILE`: read options from file
`-save_config FILE`: save the current options to file